package.json

```diff
@@ -1,9 +1,17 @@
 {
   "name": "@aligent/ts-code-standards",
-  "main": "src/index.js",
   "type": "module",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.mjs"
+    },
+    "./tsconfigs-base": "./tsconfigs/base.json",
+    "./tsconfigs-react": "./tsconfigs/react.json"
```

Comment:
@kai-nguyen-aligent Will the old import style still work. e.g. If I have `@aligent/ts-code-standards/tsconfigs/react.json` in my project, will it still work.

If not this is technically a breaking, non-backwards-compatible change so would mean bumping the major version number which would be good to avoid if possible.

Reply:
@styler3 Unfortunately not with this export. We can change the export like so to make it backward compatible:
```json
"./tsconfigs/base.json": "./tsconfigs/base.json",
"./tsconfigs/react.json": "./tsconfigs/react.json"
```
Let me update the code.

Reply:
@kai-nguyen-aligent Maybe we could do both? That way new projects can use your neater version but it won't break old projects?